targets/extension

```diff
@@ -19,7 +19,7 @@ fi
 . "${TARGETSDIR:="$PWD"}/common"
 
 ### Append to prepare.sh:
-install x11-utils xclip
+install gentoo=x11-misc/xclip,xclip debian=x11-utils, gentoo=xdpyinfo, gentoo=xdriinfo, gentoo=xprop,
```

Comment:
you could probably combine one of these with the x11-utils selector: `gentoo=xprop,x11-utils`